frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -342,9 +353,63 @@ function getMaybeMinCutItem(
   );
 }
 
+function getMaybeMeshItems(
+  maybeClickedMeshId: number | null | undefined,
+  maybeMeshIntersectionPosition: Vector3 | null | undefined,
+  visibleSegmentationLayer: APIDataLayer | null | undefined,
+  removeMesh: (segmentationLayerName: string, meshId: number) => void,
+  hideMesh: (segmentationLayerName: string, meshId: number) => void,
+  setPosition: (position: Vector3) => void,
+  refreshMesh: (segmentationLayerName: string, meshId: number) => void,
+) {
+  if (
+    maybeClickedMeshId == null ||
+    maybeMeshIntersectionPosition == null ||
+    visibleSegmentationLayer == null
+  ) {
+    return null;
+  }
+
+  return [
+    <Menu.Item
+      key="hide-mesh"
+      onClick={() => hideMesh(visibleSegmentationLayer.name, maybeClickedMeshId)}
+    >
+      Hide Mesh
+    </Menu.Item>,
+    <Menu.Item
+      key="reload-mesh"
+      onClick={() => refreshMesh(visibleSegmentationLayer.name, maybeClickedMeshId)}
+    >
+      Reload Mesh
+    </Menu.Item>,
+    <Menu.Item
+      key="jump-to-mesh"
+      onClick={() => {
+        const storeState = Store.getState();
+        const datasetScale = storeState.dataset.dataSource.scale;
```

Comment:
would it be easy to already receive the unscaled position here so that this doesn't need to happen here? I think, this division is done somewhere else, too. So, it could by DRYed a bit. However, I'm not sure whether the scale is easily known "further up".

Reply:
Yeah, the scale is easily available in the parent component. I just overlooked it 🙈